src/lib/y2storage/lvm_vg.rb

```diff
@@ -117,6 +123,22 @@ class LvmVg < Device
     #   @param lvm_lv [LvmLv] volume to delete
     storage_forward :delete_lvm_lv
 
+    # @!method max_size_for_lvm_lv(lv_type)
+    #   Returns the max size for a new logical volume of type lv_type. The size may
+    #   be limited by other parameters  (e.g. the filesystem on it). The max size also
+    #   depends on parameters like the chunk size for thin pools.
+    #
+    #   @param lv_type [LvType]
+    #   @return [DiskSize]
+    storage_forward :max_size_for_lvm_lv, as: "DiskSize"
+
+    # @!method overcommitted?
+    #   Checks whether the volume group is overcommitted. If it is, Storage#commit
```

Comment:
I guess you meant `StorageManager#commit` here. Or, at most, `Storage::Storage#commit`